carbon-p2-plugin/src/main/java/org/wso2/maven/p2/Category.java

```diff
@@ -65,17 +66,17 @@
 		return features;
 	}
 	
-    public ArrayList<CatFeature> getProcessedFeatures(MavenProject project, ArtifactFactory artifactFactory, List remoteRepositories, ArtifactRepository localRepository, ArtifactResolver resolver) throws MojoExecutionException{
+    public ArrayList<CatFeature> getProcessedFeatures(MavenProject project) throws MojoExecutionException{
```

Comment:
Currently the category section is not touched. So the code related to categories are left intact.